connect/runtime/src/test/java/org/apache/kafka/connect/runtime/health/ConnectClusterStateImplTest.java

```diff
@@ -66,6 +72,28 @@ public Void answer() {
         assertEquals(expectedConnectors, connectClusterState.connectors());
     }
 
+    @Test
+    public void connectorConfig() {
+        final String connName = "sink6";
+        final Map<String, String> expectedConfig = Collections.singletonMap("key", "value");
+        Capture<Callback<Map<String, String>>> callback = EasyMock.newCapture();
+        herder.connectorConfig(EasyMock.eq(connName), EasyMock.capture(callback));
+        EasyMock.expectLastCall().andAnswer(new IAnswer<Void>() {
```

Comment:
should we start writing those as lambdas for the new code (the code that is not meant to be backported prior to 5.0?)

Reply:
I wouldn't mind altering these to be lambdas but it'd be better if we could backport that change to 2.0 to avoid any merge conflicts down the road if a bug fix is discovered that needs to be backported that far as well. Since this PR is targeted at 2.3 I think we should hold off on making those changes here.

Reply:
I was referring only to the newly introduced methods and their tests. Not the whole class. Not a big issue.

Reply:
I think introducing mixed style here would just cause minor confusion--why use lambdas for some test cases but not for others?